app/src/androidTest/java/woowacourse/payments/ui/CardOwnerNameTextFieldTest.kt

```diff
@@ -0,0 +1,47 @@
+package woowacourse.payments.ui
+
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performTextInput
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class CardOwnerNameTextFieldTest {
+    @get:Rule
+    val composeRule = createComposeRule()
+
+    @Before
+    fun setUp() {
+        composeRule.setContent {
+            CardOwnerNameTextField()
+        }
+    }
+
+    @Test
+    fun `카드_소유자_이름의_길이는_30자를_넘을_수_없다`() {
+        // when
+        composeRule.onNodeWithText("")
+            .performTextInput("모".repeat(30))
+
+        composeRule.onNodeWithText("모".repeat(30))
+            .performTextInput("모찌")
```

Comment:
```suggestion
            .performTextInput("모".repeat(31))
```

제안한 코드처럼 한번에 제한된 길이를 넘켜서 입력하지 않은 이유가 있나용?

Reply:
저의 코드를 수정하기 전 예전코드에서 
```if(newLength <= 30) ``` 일때만, 입력이 되게 했습니다. 

테스트를 돌리니, "모"가 31가 한번에 입력되면서 테스트 코드가 통과가 안되어서 모를 30번째 입력하고 추가로 한번 더 입력했습니다! 하지만, 지금은 `take`로 가져오니 메다가 제안한 방식으로 다시 수정했습니당!

[test: 카드_소유자_이름의_길이가_30자를_넘을_경우_30자로_제한하는_테스트_수정](https://github.com/woowacourse/android-payments/pull/20/commits/5675ac9513a948af02af5a37d1fc7e5a954d22c4)